### PR TITLE
feat: Show what's next options in chat text

### PR DESCRIPTION
**Pull request by mvachhar**
Add a static chat "what's next" message to the start of the chat log for newly deployed apps.

* Chat message appears when there are fewer than 25 other messages
* Chat explains what to try after the starter deploys the chat app.
* Chat message looks as follows

image

### CODE REVIEW

File: public/client.js [hunk 1]
```diff
@@ -235,6 +236,41 @@ const showChat = async () => {
     }
   });
   
+  /* REMOVE MESSAGE BEGIN */
+  const chat = document.querySelector('.chat');
+  console.log(messages.data.length);
```
Comment (mterrel) [inline public/client.js]: Remove debug

File: public/client.js [hunk 2]
```diff
@@ -235,6 +236,41 @@ const showChat = async () => {
...
+  if(chat && messages.data.length < 25) {
+    chat.innerHTML += `<div class="message flex flex-row">
+      <img src="https://adaptable.io/img/color logo.svg" alt="Adaptable.io logo" class="avatar">
+      <div class="message-wrapper">
+      <p class="message-header">
```
Comment (mterrel) [inline public/client.js]: Indentation seems off here and with the `<ul>` tag. Maybe elsewhere too.

File: public/client.js [hunk 3]
```diff
@@ -235,6 +236,41 @@ const showChat = async () => {
...
+          Congratulations on logging in to your chat starter app.
+          <img src="https://adaptable.io/img/party-popper.svg" width="20em">
+          <br />
+          Wondering what's next? Go to the <a target="_blank" href="https://adaptable.io/docs/starters/feathers-chat-starter#what-s-next">starter guide</a> to find how to:
+          <ul>
+          <li>
+            <a target="_blank" href="https://adaptable.io/docs/starters/feathers-chat-starter#idea-2-remove-the-congratulations-message">Remove this message and the one from the login page</a>,
+          </li>
+          <li>
+            <a target="_blank" href="https://adaptable.io/docs/starters/feathers-chat-starter#idea-3-enable-github-authentication">Enable GitHub Authentication for the chat app<a href="">,
```
Comment (mterrel) [inline public/client.js]: There's an extra `<a>` tag here?

File: public/client.js [hunk 4]
```diff
@@ -235,6 +236,41 @@ const showChat = async () => {
...
+          </li>
+          <li>
+          <a target="_blank" href="https://adaptable.io/docs/starters/feathers-chat-starter#idea-3-enable-github-authentication">Add new api services to the Feathers.js backend for this app</a>.
```
Comment (mterrel) [inline public/client.js]: This should probably point to the anchor for idea 4, not idea 3.

File: public/client.js [hunk 5]
```diff
@@ -235,6 +236,41 @@ const showChat = async () => {
...
+          </li>
```
Comment (mterrel) [inline public/client.js]: Also, capitalize "API"

**Review by mterrel — CHANGES_REQUESTED** [review]

File: public/client.js (hunk 1 above)
Comment (mvachhar) [reply public/client.js]: Whoops, good catch.

File: public/client.js (hunk 3 above)
Comment (mvachhar) [reply public/client.js]: Very strange, not sure what caused this.  Also fixed.

File: public/client.js (hunk 4 above)
Comment (mvachhar) [reply public/client.js]: Fixed.

File: public/client.js (hunk 5 above)
Comment (mvachhar) [reply public/client.js]: Fixed.

File: public/client.js (hunk 2 above)
Comment (mvachhar) [reply public/client.js]: Does this indentation look better.  Man I cannot believe how dependent I am on an autoformatter :)

Comment (mvachhar) [reply public/client.js]: Fixed line 245 as well.

**Review by mterrel — APPROVED** [review]

**Pull request merged**